packages/datadog-plugin-pg/test/index.spec.js

```diff
@@ -112,6 +112,28 @@ describe('Plugin', () => {
             })
           })
 
+          if (version === '4.0.0') {
+            it('should handle error event', done => {
```

Comment:
We should be testing the issue as reported in #1034 and not only explicitly this error, otherwise we have no guarantee that the original problem is fixed and we are not catching regressions.

Reply:
If the interface changed to a promise, it should be able to test this by using the promise interface starting from the version where the interface was changed.